.github/workflows/tag-publish.yml

```diff
@@ -1,121 +1,80 @@
-          -e "VAULT_ADDR=${VAULT_ADDR}" \
-          vault:1.1.0 \
-          vault read -field username ${ARTIFACTORY_ACCOUNT_PATH})
-        ARTIFACTORY_PASSWORD=$(docker run --rm --cap-add IPC_LOCK \
-          -e "VAULT_TOKEN=${{ steps.vault-token-step.outputs.vault-token }}" \
-          -e "VAULT_ADDR=${VAULT_ADDR}" \
-          vault:1.1.0 \
-          vault read -field password ${ARTIFACTORY_ACCOUNT_PATH})
-        GCR_EMAIL=$(docker run --rm --cap-add IPC_LOCK \
-          -e "VAULT_TOKEN=${{ steps.vault-token-step.outputs.vault-token }}" \
-          -e "VAULT_ADDR=${VAULT_ADDR}" \
-          vault:1.1.0 \
-          vault read -field ci-gcr-sa-email ${VAULT_PATH_GCR})
-        GCR_KEY=$(docker run --rm --cap-add IPC_LOCK \
-          -e "VAULT_TOKEN=${{ steps.vault-token-step.outputs.vault-token }}" \
-          -e "VAULT_ADDR=${VAULT_ADDR}" \
-          vault:1.1.0 \
-          vault read -field ci-gcr-sa-key ${VAULT_PATH_GCR})
-        echo ::set-output name=gcr-email::$GCR_EMAIL
-        echo ::add-mask::$GCR_EMAIL
-        echo ::set-output name=gcr-key::$GCR_KEY
-        echo ::add-mask::$GCR_KEY
-        echo ::set-output name=artifactory-username::$ARTIFACTORY_USERNAME
-        echo ::add-mask::$ARTIFACTORY_USERNAME
-        echo ::set-output name=artifactory-password::$ARTIFACTORY_PASSWORD
-        echo ::add-mask::$ARTIFACTORY_PASSWORD
-    - name: "Bump the tag to a new version"
-      uses: databiosphere/github-actions/actions/bumper@v0.0.3
+        ref: ${{ steps.controls.outputs.checkout-branch }}
+        token: ${{ secrets.REPO_ACCESS_TOKEN }}
+    - name: Bump the tag to a new version
+      uses: databiosphere/github-actions/actions/bumper@-0.0.3
       id: tag
       env:
-        GITHUB_TOKEN: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
-        DEFAULT_BUMP: patch${{ steps.semver.outputs.semver-part }}
+        GITHUB_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+        DEFAULT_BUMP: patch
+        OVERRIDE_BUMP: ${{ steps.controls.outputs.semver-part }}
         RELEASE_BRANCHES: dev
+        HOTFIX_BRANCHES: hotfix.*
         VERSION_FILE_PATH: build.gradle
-        VERSION_LINE_MATCH: ^version
+        VERSION_LINE_MATCH: "^\\s*version = \".*\""
         VERSION_SUFFIX: SNAPSHOT
-    - name: "Publish to Artifactory"
-      run: ./gradlew artifactoryPublish
-      env:
-        ARTIFACTORY_USERNAME: ${{ steps.vault-secret-step.outputs.artifactory-username }}
-        ARTIFACTORY_PASSWORD: ${{ steps.vault-secret-step.outputs.artifactory-password }}
-        ARTIFACTORY_REPO_KEY: "libs-snapshot-local"
-    - name: Auth to GCR
-      uses: google-github-actions/setup-gcloud@master
-      with:
-        version: '270.0.0'
-        service_account_email: ${{ steps.vault-secret-step.outputs.gcr-email }}
-        service_account_key: ${{ steps.vault-secret-step.outputs.gcr-key }}
-    - name: Explicitly auth Docker for GCR
-      run: gcloud auth configure-docker --quiet
     - name: Set up AdoptOpenJDK 11
```

Comment:
I'd leave out the version number in the step name since it's clear in the `with` statement.